pkg/controller/populators/import-populator.go

```diff
@@ -181,21 +187,35 @@ func (r *ImportPopulatorReconciler) updatePVCForPopulation(pvc *corev1.Persisten
 	annotations[cc.AnnSource] = cc.SourceNone
 }
 
+func updateVddkAnnotations(pvc, pvcPrime *corev1.PersistentVolumeClaim) {
+	if cc.GetSource(pvcPrime) != cc.SourceVDDK {
+		return
+	}
+	if vddkHost := pvcPrime.Annotations[cc.AnnVddkHostConnection]; vddkHost != "" {
+		cc.AddAnnotation(pvc, cc.AnnVddkHostConnection, vddkHost)
+	}
+	if vddkVersion := pvcPrime.Annotations[cc.AnnVddkVersion]; vddkVersion != "" {
+		cc.AddAnnotation(pvc, cc.AnnVddkVersion, vddkVersion)
+	}
+}
+
+func (r *ImportPopulatorReconciler) updateImportAnnotations(pvc, pvcPrime *corev1.PersistentVolumeClaim) {
+	phase := pvcPrime.Annotations[cc.AnnPodPhase]
+	if err := r.updateImportProgress(phase, pvc, pvcPrime); err != nil {
+		r.log.Error(err, "Failed to update import progress for pvc %s/%s", pvc.Namespace, pvc.Name)
```

Comment:
why just log the error?

Reply:
I dont want to prevent the phase update in case there is some issue with the progress pod